packages/radix-ui-themes/src/components/callout.tsx

```diff
@@ -92,6 +95,7 @@ const CalloutText = React.forwardRef<CalloutTextElement, CalloutTextProps>(
         color={color}
         highContrast={highContrast}
         {...props}
+        asChild={false}
```

Comment:
Oh I think I've realised why, it's because it's already removed by `ExtractPropsForTag` right?

That's not super obvious when reviewing, I wonder if we could do something about that? Rename the type helper maybe?

Reply:
> Oh I think I've realised why, it's because it's already removed by `ExtractPropsForTag` right?

Yep. Do you have a better name in mind?